.github/workflows/test-backends.yml

```diff
@@ -64,7 +64,11 @@ jobs:
 
     strategy:
       matrix:
-        runner: ["gfx908"]
+        runner: ["rocm"]
+
+    container:
+      image: rocm/pytorch:rocm5.7_ubuntu22.04_py3.10_pytorch_2.0.1
```

Comment:
Note: when we get rid of full rocm dependency (and only depend on HSA), we will only need the driver installed and probably won't need this full image